index.html

```diff
@@ -289,7 +289,114 @@ <h2>About me</h2>
           </ul>
         </div>
       </div>
+       <div id="contact">
+        <h2>
+          I'm always interested in hearing about new projects, so if you'd like
+          to chat please get in touch.
+        </h2>
+        <form  id ="form" action="https://formspree.io/f/mwkakrqj" method="GET">
+          <input
+            type="text"
+            name="fullname"
+            id="fullname"
+            placeholder="Full name"
+            required
+          />
+          <div class="name-form">
+            <input
+              type="text"
+              name="name"
+              id="name"
+              placeholder="First name"
+              required
+            />
+            <input
+              type="text"
+              name="surname"
```

Comment:
According to our Figma design, we have `Firstname`, and `Lastname` fields only, so make sure to replace this with `lastname`.
![image](https://user-images.githubusercontent.com/34779928/190401901-bc52f07d-c111-43cc-bfd3-bbaf14751e7f.png)